dagmc/dagnav.py

```diff
@@ -289,10 +300,52 @@ def __init__(self, model: DAGModel, handle: np.uint64):
         super().__init__(model, handle)
         self._check_category_and_dimension()
 
-    def get_volumes(self):
+    @property
+    def surf_sense(self) -> list[np.uint64]:
+        """Surface sense data."""
+        return self.model.mb.tag_get_data(
+            self.model.surf_sense_tag, self.handle, flat=True
+        )
+
+    @surf_sense.setter
+    def surf_sense(self, sense_data: list[np.uint64]):
+        sense_data = [np.uint64(x) for x in sense_data]
+        self._tag_set_data(self.model.surf_sense_tag, sense_data)
```

Comment:
When we establish this relationship with the new volumes via the sense tag, we should also update the surface's parent-child relationships as well.

Reply:
Just updated this to add the parent-child relationships -- I think I did this correctly but would appreciate you double-checking